FSQCellManifest/FSQCellManifest.h

```diff
@@ -449,32 +450,32 @@ extern const NSInteger kRowIndexForFooterIndexPaths;
  
  @return A new table view cell manifest.
  */
-- (instancetype)initWithDelegate:(id)delegate 
-                         plugins:(NSArray *)plugins
+- (instancetype)initWithDelegate:(nullable id)delegate
```

Comment:
willConfigureCell is too long so it looks a little weird:
- (void)manifest:(FSQCellManifest *)manifest
willConfigureCell:(id)cell
       withModel:(id)model
     atIndexPath:(NSIndexPath *)indexPath
          record:(FSQCellRecord *)cellRecord;